### DISCUSSION TITLE
Providing urls in response data

### DISCUSSION THREAD

**Discussion by CEB-21** [tidal-music/discussions#33] [Ideas & Feature requests]
I think it's great that you all provide the ids of artist, albums, and tracks but what would really be useful is also providing urls for each of them as you would find in other popular apis like Spotify and Apple Music.

**Comment by stefanmoro** [comment]
Hey @CEB-21, and thank you for the proposal.

I agree, adding a preformatted URL to the resource (track, video, album, artist etc) in the corresponding response sounds like a good idea. 

Let us investigate a bit and see what we can do about it. We'll get back to you.

**Reply by CEB-21** [reply]
Thanks for getting the ball rolling on this one, @stefanmoro!

**Comment by aliaksei-taliuk** [comment]
@CEB-21 we happy to announce that we've added a new property called tidalUrl to all our response models (album, artist, track, video). Thanks for participation and your input!
Best regards

**Reply by CEB-21** [reply]
You guys are amazing and work lighting fast! I can't wait to incorporate this into my app! THANK YOU!
